src/components/AdminHeader.tsx

```diff
@@ -1,14 +1,35 @@
 import Link from "next/link";
 import Stack from "react-bootstrap/Stack";
+import KintaiAdmin from "@/api/KintaiAdmin";
+import { useEffect, useState } from "react";
+import { GetAdmins200ResponseDataInner } from "@/api/typescript-axios";
```

Comment:
型名が長くてわかりにくいので、別名をつけるのはどうですかね？

```suggestion
import { GetAdmins200ResponseDataInner as Admin } from "@/api/typescript-axios";
```

Reply:
ありがとうございます！
修正しました！
https://github.com/tatata05/next-app/pull/20/commits/d00375b38c20cbdb0f2e33c02931ef0fa004160a